TreasureHunt/Assets/2.Scripts/MakerSceneManager.cs

```diff
@@ -16,22 +16,21 @@ void start()
 
 	public void uploadToCloud()
 	{
-		/*TODO(hyunSo): works on my laptop but not on my phone. Guess I should have change the camera methad as WebCam
```

Comment:
I notice you've deleted this TODO. Does that mean it works now? I don't see the camera method being changed out.